### DISCUSSION TITLE
After starting remote control, the current window of the remote computer will lose focus for a few seconds

### DISCUSSION THREAD

**Discussion by hudingwen** [rustdesk/rustdesk#9187] [General]
### Bug Description

After starting remote control, the current window of the remote computer will lose focus for a few seconds

### How to Reproduce

For example, when my remote computer is performing some operations, if I initiate remote control of the remote computer, the remote pop-up prompt in the upper right corner will interrupt the task my remote computer is performing for a few seconds. Is it a program problem or a problem with my settings?

### Expected Behavior

Although there is a pop-up remote reminder, it should not interrupt the actions being performed by my remote computer.

### Operating system(s) on local side and remote side

All -> WIndow 10

### RustDesk Version(s) on local side and remote side

1.3.0 -> 1.3.0

### Screenshots

![1724743277975](https://github.com/user-attachments/assets/b3b27b28-5c5b-4085-be76-6a95dc412271)

**Comment by jjolmo** [comment]
I think this is happening to me too. But I'm not sure if the issue is related

For a few seconds I can write on apps in my Debian machine. Then, the window just loses focus, making impossible to write any simple word or command
